cypress/support/step_definitions/event.js

```diff
@@ -36,12 +36,12 @@ When('I drag {string} onto {string}', (templateOriginSelector, templateDestinati
 
 When('I select {string} in {string}', (option, templateSelector) => {
   const selector = nunjucks.renderString(templateSelector, cy.context);
+  // eslint-disable-next-line cypress/unsafe-to-chain-command,cypress/no-unnecessary-waiting
   cy.get(selector)
-    .click();
-  cy.get(option)
-    .click();
-  cy.get(selector)
-    .click();
+    .click()
+    .get(option)
+    .click()
+    .wait(500);
```

Comment:
try without disabling eslint
(see `cypress/support/step.../html.js line 80-83)

Reply:
Done except for eslint waiting warning